lib/versioner/simple.go

```diff
@@ -19,9 +19,10 @@ func init() {
 }
 
 type simple struct {
-	keep       int
-	folderFs   fs.Filesystem
-	versionsFs fs.Filesystem
+	keep            int
+	folderFs        fs.Filesystem
+	versionsFs      fs.Filesystem
+	copyRangeMethod fs.CopyRangeMethod
```

Comment:
This seems like it could just obey the folder copyrangemethod?

Reply:
I did that at first, but it was ugly pushing it down singular values from the folder. The alternative could be to push down whole folder config into the versioner factory method?

Reply:
Yeah, I guess try that. It would be odd to enable fancy copyranges on a folder and not gain the advantage on versioning automatically.